### PR TITLE
Remove OpenXmlPackage.Close and all references

### PR DESCRIPTION
**Pull request by tomjebo**

### CODE REVIEW

File: test/DocumentFormat.OpenXml.Tests/CreateFromTemplateTests.cs [hunk 1]
```diff
@@ -20,7 +20,7 @@ public void CanCreatePresentationFromTemplate()
                 var part = packageDocument.PresentationPart;
                 var root = part.Presentation;
 
-                packageDocument.SaveAs(Path.GetTempFileName()).Close();
+                using var temp = packageDocument.SaveAs(Path.GetTempFileName());
```
Comment (twsouthwick) [inline test/DocumentFormat.OpenXml.Tests/CreateFromTemplateTests.cs:23]:
```suggestion
                using var _= packageDocument.SaveAs(Path.GetTempFileName());
```

File: test/DocumentFormat.OpenXml.Tests/CreateFromTemplateTests.cs [hunk 2]
```diff
@@ -36,7 +36,7 @@ public void CanCreateSpreadsheetFromTemplate()
                 var part = packageDocument.WorkbookPart;
                 var root = part.Workbook;
 
-                packageDocument.SaveAs(Path.GetTempFileName()).Close();
+                using var temp = packageDocument.SaveAs(Path.GetTempFileName());
```
Comment (twsouthwick) [inline test/DocumentFormat.OpenXml.Tests/CreateFromTemplateTests.cs:39]:
```suggestion
                using var _= packageDocument.SaveAs(Path.GetTempFileName());
```

File: test/DocumentFormat.OpenXml.Tests/CreateFromTemplateTests.cs [hunk 3]
```diff
@@ -52,7 +52,7 @@ public void CanCreateWordprocessingDocumentFromTemplate()
                 var part = packageDocument.MainDocumentPart;
                 var root = part.Document;
 
-                packageDocument.SaveAs(Path.GetTempFileName()).Close();
+                using var temp = packageDocument.SaveAs(Path.GetTempFileName());
```
Comment (twsouthwick) [inline test/DocumentFormat.OpenXml.Tests/CreateFromTemplateTests.cs:55]:
```suggestion
                using var _= packageDocument.SaveAs(Path.GetTempFileName());
```

File: test/DocumentFormat.OpenXml.Tests/SaveAndCloneTests.cs [hunk 4]
```diff
@@ -181,7 +186,7 @@ public void CanWildlyCloneAndFlush()
 
                         using (var tempFile = TemporaryFile.Create())
                         {
-                            clone.SaveAs(tempFile.Path).Close();
+                            using var temp = clone.SaveAs(tempFile.Path);
```
Comment (twsouthwick) [inline test/DocumentFormat.OpenXml.Tests/SaveAndCloneTests.cs:189]:
```suggestion
                            using var _= clone.SaveAs(tempFile.Path);
```

File: test/DocumentFormat.OpenXml.Tests/SaveAndCloneTests.cs [hunk 5]
```diff
@@ -197,7 +202,7 @@ public void CanDoPackageBasedCloningWord()
             {
                 using (var package = Package.Open(ms, FileMode.Create))
                 {
-                    source.Clone(package).Close();
+                    using var temp = source.Clone(package);
```
Comment (twsouthwick) [inline test/DocumentFormat.OpenXml.Tests/SaveAndCloneTests.cs:205]:
```suggestion
                    using var _ = source.Clone(package);
```

File: test/DocumentFormat.OpenXml.Tests/SaveAndCloneTests.cs [hunk 6]
```diff
@@ -218,7 +223,7 @@ public void CanDoPackageBasedCloningSpreadsheet()
...
```
Comment (twsouthwick) [inline test/DocumentFormat.OpenXml.Tests/SaveAndCloneTests.cs:226]:
```suggestion
                    using var _ = source.Clone(package);
```

File: test/DocumentFormat.OpenXml.Tests/SaveAndCloneTests.cs [hunk 7]
```diff
@@ -239,7 +244,7 @@ public void CanDoPackageBasedCloningPowerpoint()
...
```
Comment (twsouthwick) [inline test/DocumentFormat.OpenXml.Tests/SaveAndCloneTests.cs:247]:
```suggestion
                    using var _ = source.Clone(package);
```

**Review by twsouthwick — DISMISSED** [review]
You can use a discard here.

A lot of these are for SaveAs. This seems like an awkward thing to use...

**Review by twsouthwick — APPROVED** [review]

**Comment by tomjebo** [comment]
@twsouthwick the pipeline build ran successfully on these changes but for some reason didn't report back to this PR on github. Can you just do a quick re-review?

**Pull request merged**